src/extension/content/convert/Retrieve.re

```diff
@@ -7,12 +7,12 @@ type typeTable = {el: Element.t, text: string, remove: unit => unit};
 type listing = {els: list Element.t, text: string, replace: string => Element.t};
 
 let getTypeTable pre =>
-  switch (Js.Null.to_opt (Element.nextSibling pre)) {
+  switch (Js.Null.to_opt (Element.nextElementSibling pre)) {
   | None => None
   | Some el =>
     if (Js.to_bool (DOMTokenList.contains (Element.classList el) "typetable")) {
       let text = Element.innerText el;
-      switch (Js.Null.to_opt (Element.nextSibling el)) {
+      switch (Js.Null.to_opt (Element.nextElementSibling el)) {
       | None => Some {el, text, remove: fun () => Element.remove el}
       | Some next =>
         if (Node.nodeType el == Node._TEXT_NODE) {
```

Comment:
It looks like we are explicitly handling text nodes here - so I feel like this will likely break in other cases. (I can't find it right now but I'm almost sure that there are some ocamldoc pages which put part of their typetable into text nodes.) Is there another fix?